.github/workflows/win.yml

```diff
@@ -0,0 +1,114 @@
+name: Build Moveit Core on all platforms
+
+on: push
+
+jobs:
+  run_moveit_core_compilation:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
```

Comment:
Since this runs on more than windows, maybe the file should be named something other than `win.yml`?